package/etc/conf.d/conflib/syslog/app-github_ent.conf

```diff
@@ -0,0 +1,20 @@
+block parser github_ent-parser() {    
+ channel {
+        rewrite {
+            r_set_splunk_dest_default(
+                index("gitops")
+                source("github:enterprise:audit")
+                sourcetype('github:enterprise:audit')
+                vendor_product("github_enterprise")
```

Comment:
this should match the key in docs GITHUB_ENT (lower case)

Reply:
Yes I have made the change.